.build/install.ps1

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/pwsh
+
+If ($isLinux) {
+    $env:DEBIAN_FRONTEND='noninteractive'
+    lsb_release -irc
+}
+
+###################
+## Setup PostGIS ##
+###################
+
+If ($isWindows) {
+    If (!(Test-Path $env:POSTGIS_EXE)) {
+        Write-Host 'Downloading PostGIS'
+        (New-Object Net.WebClient).DownloadFile("http://download.osgeo.org/postgis/windows/pg10/$env:POSTGIS_EXE", "$env:POSTGIS_EXE")
+    }
+    Write-Host 'Installing PostGIS'
+    iex ".\$env:POSTGIS_EXE /S /D='C:\Program Files\PostgreSQL\10'"
+}
+If ($isLinux) {
```

Comment:
That's actually the feed we pull PostGIS 2.4 from, since Xenial only has 2.2 in the standard repos for now.

At least that's my understanding, are you seeing something different?